invader_payment_paypal/services/payment_paypal.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2019 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+import requests
+from cerberus import Validator
+from odoo import _
+from odoo.addons.base_rest.components.service import to_int
+from odoo.addons.component.core import AbstractComponent
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+PAYPAL_ERROR = {"invalid_client": _("Invalid Paypal Credential")}
+
+PAYPAL_STATUS = {
+    "CREATED": "draft",
+    "SAVED": "draft",
+    "APPROVED": "pending",
+    "VOIDED": "error",
+    "COMPLETED": "done",
+}
+
+
+class PaymentServicePaypal(AbstractComponent):
+
+    _name = "payment.service.paypal"
```

Comment:
@sebastienbeau Can you put the service method as first method declared into the service to improve readability